.github/workflows/proto.yml

```diff
@@ -40,3 +44,14 @@ jobs:
         run: make proto-check-breaking PR_TARGET_REPO=https://github.com/${{ github.repository }}.git PR_TARGET_BRANCH=${{ github.base_ref }}
         working-directory: golang/cosmos
         if: env.GIT_DIFF
+  up-to-date:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - name: check protobuf definitions are up-to-date
+        run: |
+          cd packages/cosmic-proto
+          ./update-protos.sh
```

Comment:
The `update-protos.sh` on master is a little too dependent on JS.  If you change that file's `make all` into `make go-mod-cache`, it will only have Golang dependencies and you should be good.